### PR TITLE
Add older changes to the CHANGES file

### PR DESCRIPTION
**Pull request by timopollmeier**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -14,17 +14,30 @@ Main changes compared to gvm 8.0+beta2:
...
 * Missing data in credentials no longer prevents slave tasks from starting.
   Instead the scan will start without the credential.
+* An issue preventing "Start Task" alerts to run has been fixed.
 * Handling of failed/successful SNMP Authentication has been added to the
-  HTML and LaTeX report formats.
+  HTML, LaTeX and PDF report formats.
 * A new password-only credential type has been added
 * The Sourcefire alert now accepts a password credential for PKCS12 decryption.
+* The source code and GMP documentation have been cleaned up.
+* A seciton about deprecated GMP elements has been added to the documentation.
```
Comment (cfi-gb) [inline CHANGES]: seciton -> section

File: CHANGES [hunk 2]
```diff
@@ -14,17 +14,30 @@ Main changes compared to gvm 8.0+beta2:
...
+* CREATE_TASK now requires a name.
+* The compile-time LOG option has been removed.
+* The `--modify-scanner` option now also accepts UNIX sockets.
+* Support for report content composition has been added.
+* TEST_ALERT now also works if NVTs are missing.
+* LSC errors are now logged as warnings.
 * Remediation support has been added (GMP CREATE_TICKET, GET_TICKETS, etc).
 * Missing data in credentials no longer prevents slave tasks from starting.
   Instead the scan will start without the credential.
+* An issue preventing "Start Task" alerts to run has been fixed.
```
Comment (mattmundell) [inline CHANGES]:
```suggestion
* An issue preventing "Start Task" alerts from running has been fixed.
```

File: CHANGES (hunk 1 above)
Comment (timopollmeier) [reply CHANGES]: Fixed in ea0572edc82e0dcac054e1d4b25e4fa712c5677b.

File: CHANGES (hunk 2 above)
Comment (timopollmeier) [reply CHANGES]: Fixed in ea0572edc82e0dcac054e1d4b25e4fa712c5677b.

**Pull request merged**